foodsaving/stores/models.py

```diff
@@ -30,6 +30,13 @@ def __str__(self):
         return '{} ({})'.format(self.name, self.group)
 
 
+class Feedback(BaseModel):
+    given_by = models.ForeignKey('users.User', on_delete=models.CASCADE, related_name='user_feedback')
+    about = models.ForeignKey('PickupDate')
```

Comment:
I find the name `about` unintuitive, and it sounds like it's maybe a generic relation (but it's not) - how about `pickup`?

Reply:
I'm ok with both, change it if you agree with Nick and you feel like doing refactoring.

Reply:
we thought the same. `about` is too unintuitive. we will change it then.

Reply:
And are you sure that the name `pickup` will not crash with `pickup` at different places in code?